.github/workflows/git-town.yml

```diff
@@ -0,0 +1,19 @@
+name: Git Town
```

Comment:
Can you add more color to the name? I'm not sure contributors will know what git town is

Reply:
seems to be failing?

Reply:
Yea, i'm not really sure what's going on: Google search didn't find anything. Maybe it'll work in real life? If not, we can always revert it